rose/web/game.js

```diff
@@ -405,6 +403,38 @@ var ROSE = (function() {
         }
     }
 
+    function SoundController(loader) {
+        this.loader = loader;
+
+        var localStorageMute = localStorage.getItem('muted');
+        this.muted = (typeof localStorageMute !== 'undefined') ? localStorageMute == "true" : false;
+
+        var icons = {
+            false: "res/icons/mute.svg",
+            true: "res/icons/unmute.svg"
+        };
+
+        var self = this;
+
+        $("#toggle_mute img").attr('src', icons[this.muted]);
+
+        loader.load("res/soundtrack/Nyan_Cat.ogg", function(sound) {
+            sound.muted = self.muted;
+            sound.loop = true;
```

Comment:
Don't we need this in the previous commit? The sound should always loop, regardless of having a way to mute it.